src/Sarif/IProcessRunner.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public interface IProcessRunner
```

Comment:
>IProcessRunner [](start = 21, length = 14)

Allows `GitHelper`, which runs an external process, to participate in unit tests.

Reply:
This is fine. I don't think I would have created an interface for this.

---
In reply to: [479482578](https://github.com/microsoft/sarif-sdk/pull/2047#discussion_r479482578) [](ancestors = 479482578)

Reply:
Changed to a delegate.

---
In reply to: [479552004](https://github.com/microsoft/sarif-sdk/pull/2047#discussion_r479552004) [](ancestors = 479552004,479482578)